Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.17.6 as builder
+
+# Clone repositories
+WORKDIR /repos
+RUN git clone https://github.com/ElrondNetwork/rosetta-docker-scripts.git --branch=rc/v1 --depth=1
```

Comment:
we should add some TODOs to change these as soon as we merge the branches and create releases

Reply:
Fixed.